src/d3d11/d3d11_context.h

```diff
@@ -7,6 +7,10 @@
 #include "d3d11_context_state.h"
 #include "d3d11_device_child.h"
 
+#ifndef WINBOOL
+typedef BOOL WINBOOL;
```

Comment:
I think `WINBOOL` shouldn't be a thing in the first place, if it's the same as `BOOL` then I think we should rather get rid of all uses of `WINBOOL` instead.